index.html

```diff
@@ -157,9 +157,9 @@ <h2 class="aparatus__title title">АВТОМАТИЗОВАНИЙ ПРОГРАМ
                 <div class="aparatus__about about-aparatus">
                   <div> <img src="img/Aparatus.png" alt="Aparatus" class="about-aparatus__image"></div>
                   <div class="about-aparatus__infoAparatus infoAparatus">
```

Comment:
Rename class to the about-aparatus__text. And here you can use only one class (there is no need to create infoAparatus, only about-aparatus__text)